test/parallel/test-fs-buffertype-writesync.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const common = require('../common');
+
+// This test insures that writeSync does support input buffers which
```

Comment:
Also, I don’t quite understand the statement here – you talk about `input buffers` for writeSync, but the only time you are actually using buffers is in the readFileSync output?